be/src/runtime/datetime_value.h

```diff
@@ -315,7 +316,7 @@ class DateTimeValue {
 
     //unix_timestamp is called with a timezone argument,
     //it returns seconds of the value of date literal since '1970-01-01 00:00:00' UTC
-    bool unix_timestamp(int64_t* timestamp, const std::string& timezone) const;
+    bool unix_timestamp(int64_t* timestamp, const std::string_view& timezone) const;
```

Comment:
How about adding a prepare function?
Then when the timezone is const value, timezone will only be find one time in prepare function.

Reply:
convert_tz already has a prepare function, the rest of the parameters in this query are not constants, so there is no way to use constant optimization